great_expectations_cloud/agent/agent.py

```diff
@@ -11,7 +11,7 @@
 from importlib.metadata import version as metadata_version
 from typing import TYPE_CHECKING, Any, Dict, Final
 
-from great_expectations import get_context
+from great_expectations import get_context  # type: ignore[attr-defined] # TODO: fix this?
```

Comment:
I'm not sure how to resolve these `attr-defined` issues--to me it looks like the imports should work. If I install 0.18.12, they do...but as soon as I install GX Core 0.18.13, they break.

Reply:
This one is confusing.
I somewhat understand the `great_expectations.__version__` one because we don't export types for that.